gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobInfo.java

```diff
@@ -81,6 +84,14 @@ public StorageObject apply(BlobInfo blobInfo) {
   private final String contentLanguage;
   private final Integer componentCount;
 
+  public static final class ImmutableEmptyMap<K, V> extends AbstractMap<K, V> {
```

Comment:
I think that adding javadoc saying that this class is meant for internal use only is enough.